src/Take.Elephant.Sql/Take.Elephant.Sql.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.1</TargetFrameworks>
     <GeneratePackageOnBuild>False</GeneratePackageOnBuild>
-    <Version>0.5.0</Version>
+    <Version>0.10.72.3</Version>
```

Comment:
I think that we avoid commiting this change. It is useful and meaningful only for local dev.

Reply:
Truth